### ISSUE TITLE
Dynamic classname prop not rerendering on DOM

### ISSUE THREAD

**Issue by bryant988** [gilbarbara/react-inlinesvg#131]
This might be related to the same issue as #48 

I have a dynamic **className** prop being passed into svg component but it does not seem to rerender it to the DOM (even though **className** prop changes) unless i set the **key** prop to include the dynamic classname.

```
const imageClass = // something dynamic

<SVG
    className={imageClass}
    style={this.props.style}
    key={`${this.props.src}_${imageClass}`}
    src={this.props.src}
    cacheRequests={true}
/>
```

Is there a better way or am I possibly doing something wrong?
Could this possibly be an issue with the other props being dynamic (title/style/etc)?

**Comment by gilbarbara** [comment]
Hey,

Please create an example in [codesandbox](https://codesandbox.io/) or similar.

**Comment by bryant988** [comment]
https://codesandbox.io/embed/react-inlinesvg-classname-wupe6

**Comment by idmadj** [comment]
Hey there, just ran into the issue myself. This is caused by the fact that props are only passed down once, when the SVG file has completed loading. Using the key prop kinda works around the problem, but doesn't allow CSS transitions to happen since a new node is mounted every time the key changes.

I've posted a potential fix. Happy to hear your thoughts @gilbarbara.

**Comment by gilbarbara** [comment]
Fixed in 1.1.7
